Cargo.toml

```diff
@@ -24,6 +24,7 @@ bitvec = "1"
 byteorder = "1"
 cesu8 = "1.1.0"
 cfb8 = "0.7"
+derive_more = "0.99.17"
```

Comment:
Would like to avoid adding this dependency if possible.

Reply:
That's what I thought.. I guess I need to write another macro then for this and wrap the enum? 😬